src/code42cli/cmds/securitydata.py

```diff
@@ -171,6 +178,12 @@ def clear_checkpoint(state, checkpoint_name):
     "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
 )
 @sdk_options()
+@click.option(
+    "--include-all",
+    default=False,
+    is_flag=True,
+    help="Display complete list of data items from the first level of the nested response.",
```

Comment:
Similarly here, I think `properties` is the right word for `data items`

Reply:
Another thing here, it is immediately clear to me what is at the first level.  Sometimes responses just have `data` at the first level.... Does this include the name of the nest objects with a value of `{ ... }` (like Chrome dev tools), or does it exclude it entirely.


Maybe it should say `Display simple properties of the primary level of the nested response`